Tools/WebKitTestRunner/TestInvocation.cpp

```diff
@@ -176,8 +178,10 @@ void TestInvocation::invoke()
 
     // FIXME: We should clear out visited links here.
 
-    WKPageSetPageZoomFactor(TestController::singleton().mainWebView()->page(), 1);
-    WKPageSetTextZoomFactor(TestController::singleton().mainWebView()->page(), 1);
+    WKPageSetPageZoomFactor(page, 1);
+    WKPageSetTextZoomFactor(page, 1);
+
+    WKPageClearOpenerForTesting(page);
```

Comment:
TestController::resetStateToConsistentValues would probably be a better place for this.  Probably several other things that are in this function, too, like resetting the zoom level.